src/js/lib.js

```diff
@@ -16,22 +16,28 @@ var lib = (function() {
   }
 
   function errorLog(context) {
+    console.log('lib.errorLog deprecated.  Find a different way to pass this error.')
```

Comment:
👍  as soon as I finish the tasks for adapting the GUI, I will. 

For now, it is still being called from there